cypress/e2e/charging_stations.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe('Charging station actions', () => {
+  beforeEach(() => {
+    cy.clearAllCookies();
+    cy.clearAllLocalStorage();
+    cy.clearAllSessionStorage();
+
+    //TODO figure out why this is thrown!
+    const stub = cy.stub();
+    Cypress.on('uncaught:exception', (err, runnable) => {
+      if (err.message.includes('ResizeObserver')) {
+        stub();
+        return false;
+      }
+    });
+
+    cy.visit('http://localhost:5173/charging-stations');
```

Comment:
we may want to enhance this I would think so that we arent hardcoding the strings everywhere... Maybe we can have a helper function or something that just takes in the path and returns the full URL, but also, ideally, the paths are also constants... Right now, I believe the paths are all just strings, but it may be good to tie those together, what do you think?

Reply:
The `localhost:port` part will be removed. So it uses the baseUrl of the`cyrpress.config.ts` 
Not sure if it's worth going down the rabbit hole of putting the urls of pages in constants. We could re-use them here then, I agree. But I don't have experience if that is really best practice.